rest-server/server/src/main/scala/com/actionml/router/http/routes/EnginesRouter.scala

```diff
@@ -109,14 +109,14 @@ class EnginesRouter(implicit inj: Injector) extends BaseRouter with Authorizatio
   private def createEngine(implicit log: LoggingAdapter): Route = asJson { engineConfig =>
     log.info("Create engine: {}", engineConfig)
     completeByValidated(StatusCodes.Created) {
-      (engineService ? CreateEngine(engineConfig.toString)).mapTo[Response]
+      (engineService ? CreateEngine(engineConfig.noSpaces)).mapTo[Response]
     }
```

Comment:
In general all bodies returned to REST are formatted to be displayed even if compression would be better. This is because they are displayed by the CLI's use of the REST API. This may make the CLI responses unreadable--not sure?

Reply:
here is the input part, we will add pretty print on the output

Reply:
this PR must include the output, if you mean the python formatted response.